test/sharness/t0112-gateway-cors.sh

```diff
@@ -141,18 +141,18 @@ test_expect_success "Assert the default API.HTTPHeaders config is empty" '
 test_expect_success "Default CORS GET to {gw}/api/v0" '
   curl -svX GET -H "Origin: https://example.com" "http://127.0.0.1:$GWAY_PORT/api/v0/cat?arg=$thash" >/dev/null 2>curl_output
 '
-test_expect_success "Default CORS GET response from {gw}/api/v0 is 403 Forbidden and has no CORS headers" '
+test_expect_success "Default CORS GET response from {gw}/api/v0 is 403 Forbidden and has regular CORS headers" '
```

Comment:
```suggestion
# HTTP 403 is returned because Kubo has additional protections on top of regular CORS,
# namely it only allows browser requests with localhost Origin header.
test_expect_success "Default CORS GET response from {gw}/api/v0 is 403 Forbidden and has regular CORS headers" '
```